WordPress/Classes/ViewRelated/Post/AbstractPostListViewController.swift

```diff
@@ -350,8 +350,10 @@ class AbstractPostListViewController: UIViewController,
 
     fileprivate func configureInitialScrollInsets() {
         if #available(iOS 11.0, *) {
```

Comment:
Can we rebase `develop` into `issues/11646-post-list-revamp`? By doing that I can remove these checks. Or better: I will have the checks removed because you already did. 🥇